### DISCUSSION TITLE
Set the addons theme to the system appearance

### DISCUSSION THREAD

**Discussion by scratchusernamemrtbts** [ScratchAddons/ScratchAddons#637] [Ideas]
It would be cool if the extension will detect the system appearance and set the theme to the system appearance but we can still turn off this function and toggle the theme by our self

**Comment by GrahamSH-LLK** [comment]
This would be better suited as a setting that is off at the start.

**Reply by scratchusernamemrtbts** [reply]
Ok

**Comment by Hans5958** [comment]
We can use `window.matchMedia("(prefers-color-scheme: dark)").matches` for that ig

**Reply by Hans5958** [reply]
This is not the answer yet. The answer will be something in the lines that someone approves and implements the idea.
